docs/dev/wrapping.rst

```diff
@@ -0,0 +1,204 @@
+
+Wrapping an Existing Library
+============================
+
+Many cosmology libraries already exist, such as ``astropy.cosmology``,
+``CLASS``, and ``CAMB``, and all have different inerfaces that are not
+compatible with each other.  The Cosmology API is designed to be a common
+interface and we can easily wrap existing codes to make them compatible with the
+Cosmology API.  Many of the common cosmology libraries are already wrapped and
+available in the `cosmology.compat
+<https://github.com/cosmology-api/cosmology-compat>`_ module.
+
+
+If there is an unsopported library that you would like to wrap, the Cosmology
+API is designed to be as simple as possible.  Essentially, you need to create an
+object that implements the :class:`cosmology.api.CosmologyWrapper` interface, as
```

Comment:
The `cosmology.api.CosmologyWrapper` class isn't currently documented anywhere. That's not a problem, I just want to point it out for a potential follow-up issue.

Reply:
Yeah, we need a section in both the user docs and dev docs about it.